### PR TITLE
Adding a new allowed value of ln=3 to FPDF.multi_cell

### PR DESCRIPTION
**Pull request by Lucas-C**
_cf._ https://github.com/PyFPDF/fpdf2/issues/63

### CODE REVIEW

**Comment by Lucas-C** [comment]
Currently the regression test `test_ln_positioning_and_page_breaking_for_multicell` fails and need to be fixed/ investigated:
https://github.com/PyFPDF/fpdf2/blob/master/test/cell_multi_cell_refactor/test_ln_positioning_and_page_breaking_for_multicell.pdf

File: fpdf/fpdf.py [hunk 1]
```diff
@@ -1050,7 +1048,8 @@ def multi_cell(
             link (str): optional link to add
             ln (int): Indicates where the current position should go after the call.
                 Possible values are: `0`: to the right ; `1`: to the beginning of the next line ;
-                `2`: below. Putting 1 is equivalent to putting 0 and calling `ln` just after.
+                `2`: below ; `3`: to the right .
```
Comment (fbernhart) [inline fpdf/fpdf.py:1051]: Is this just a copy/paste mistake or are `0` and the new value `3` behaving the same way and both are "to the right"? 🤔

Comment (Lucas-C) [reply fpdf/fpdf.py]: It was incomplete :) This is now better detailed.

**Comment by Lucas-C** [comment]
This is ready for review

**Comment by sree293** [comment]
Hello Lucas thanks for all your help and is it included in any of the major versions that can downloaded form pip, So that I can update it in my projects?

**Comment by Lucas-C** [comment]
No new release has been published for now, but you can install the latest version from this repo `master` branch with this command:

    pip install git+https://github.com/PyPDF/fpdf2.git@master

**Comment by Lucas-C** [comment]
This is now released in the latest version!

**Pull request merged**